src/main/java/com/casper/sdk/service/CasperService.java

```diff
@@ -236,6 +238,28 @@ DictionaryData getStateDictionaryItem(@JsonRpcParam("state_root_hash") String st
     @JsonRpcMethod("info_get_status")
     StatusData getStatus();
 
+
+    /**
+     * Returns an AddressableEntity from the network
+     *
+     * @param entityIdentifier entity identifier
+     * @return object holding the AddressableEntity or LegacyAccount
+     */
+    @JsonRpcMethod("state_get_entity")
+    StateEntityResult getStateEntity(@JsonRpcParam("entity_identifier") EntityIdentifier entityIdentifier);
+
+    /**
+     * Returns an AddressableEntity from the network
+     *
+     * @param entityIdentifier entity identifier
+     * @param blockIdentifier block identifier
+     * @return object holding the AddressableEntity or LegacyAccount
+     */
+    @JsonRpcMethod("state_get_entity")
+    StateEntityResult getStateEntity(@JsonRpcParam("entity_identifier") EntityIdentifier entityIdentifier,
```

Comment:
Why do you need two getStateEntity  methods isn't 'blockIdentifier' just an optional parameter that can be documented as such?

Delete the  _'StateEntityResult getStateEntity(@JsonRpcParam("entity_identifier") EntityIdentifier entityIdentifier);'_  method